superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterLabel.tsx

```diff
@@ -45,6 +45,7 @@ import {
   CustomFrame,
   AdvancedFrame,
 } from './components';
+import { DateLabel } from './components/DateLabel';
```

Comment:
kindly move the `DateLabel` into `index.ts` in the `components` so that import like above Frame. for instance:

```typescript
import {
  CommonFrame,
  CalendarFrame,
  CustomFrame,
  AdvancedFrame,
  DateLabel
} from './components';
```

Reply:
Good point, done!